nginx/lib.sls

```diff
@@ -44,8 +44,11 @@ htpasswd -bd /etc/nginx/htpasswd-{{appslug}} {{username}} {{password}}:
   #     } 
   #}
 {% macro nginx_custom_log_formats_and_files(log_config) %}
-  {% for format_name, format_json in log_config.get('formats', {}) %}log_format {{ format_name }} '{{ format_json }}';{% endfor %}
+  {% for format_name, format_json in log_config.get('formats', {}).iteritems() %}log_format {{ format_name }} '{{ format_json }}';
```

Comment:
A small thing, this template might be a bit easier to read if you use `{%-` and `-%}` delimiters to ignore leading and trailing whitespace, eg:
```
{% for format_name, format_json in log_config.get('formats', {}).iteritems() -%}
    log_format {{ format_name }} '{{ format_json }}';
{%- endfor %}
```